File: official/docs/java/shipments/LabelShipment.java
import com.easypost.EasyPost;
import java.util.HashMap;

public class LabelShipment {
    public static void main(String[] args) {
        EasyPost.apiKey = System.getenv("EASYPOST_API_KEY");
        
        Shipment shipment = Shipment.retrieve("shp_...");

        HashMap<String, Object> params = new HashMap<String, Object>();
        params.put("file_format", "ZPL");

        shipment.label(params);

        System.out.println(shipment);
    }
}

File: official/docs/csharp/shipments/LabelShipment.cs
using EasyPost;

EasyPost.ClientManager.SetCurrent("EASYPOST_API_KEY");

Shipment shipment = await Shipment.Retrieve("shp_...");

await shipment.GenerateLabel("ZPL");

Console.WriteLine(JsonConvert.SerializeObject(shipment, Formatting.Indented));
